neuralprophet/df_utils.py

```diff
@@ -266,7 +271,7 @@ def normalize(df, data_params, local_modeling=False):
     """
 
     if isinstance(df, list):
-        df_list = df.copy()
+        df_list = copy_list(df)
```

Comment:
Did not consider everyone as a list because of the merging and unmerging when considering global modeling

Reply:
Which inputs are we not considering?